charts/service/templates/policy-server/deployment.yaml

```
@@ -0,0 +1,101 @@
+  template:
+    metadata:
+      labels:
+        app: fulfillment-policy-server
+    spec:
+      volumes:
+      - name: tls
+        secret:
+          secretName: fulfillment-policy-server-tls
+      containers:
+      - name: policy-server
+        image: {{ .Values.images.service }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)